src/igvfd/schemas/analysis_set.json

```diff
@@ -106,7 +106,8 @@
         "samples.sample_terms.term_name",
         "samples.targeted_sample_term.term_name",
         "submitted_by.title",
-        "summary"
+        "summary",
+        "functional_assay_mechanisms.term_name"
```

Comment:
Move it above samples.summary, so this list is in the alphabetical order.

Reply:
will reorder it